src/main/java/com/airbnb/billow/ARNGenerator.java

```diff
@@ -33,7 +34,10 @@ private static String loadAccountNumber(AmazonIdentityManagementClient iamClient
         String userARN = null;
 
         try {
-            GetUserResult getUserResult = iamClient.getUser();
+            final String USER_NAME = "MikeCurtis";
```

Comment:
I think I get it...seems lame. Can you leave a comment about why we do this. Also it may be best not to use `USER_NAME= "MikeCurtis"`